iron-overlay-manager.html

```diff
@@ -13,199 +13,261 @@
 <script>
 
   /**
-   * @struct
-   * @constructor
+   * Singleton instance.
+   * This is compiler-friendly, because properties can be safely renamed.
+   * @typedef {Object} Polymer.IronOverlayManager
```

Comment:
Unfortunately it doesn't do any of the above. The error is thrown if we misspell a function name, but that's it :/